### ISSUE TITLE
Cannot achieve ResNet (2+1)D accuracy scores

### ISSUE THREAD

**Issue by omedivad** [pytorch/vision#1600]
Hi, I'm using code inside vision/references to evaluate r2plus1d_18 model. However I'm not able to reach @TOP1 and @TOP5 accuracy scores shown in [torchvision website](https://pytorch.org/docs/stable/torchvision/models.html#video-classification).

I have 19761 validation video, one of which is corrupted and other 2 are not long enough to extract one clip 16x112x112.
I'm using tensor dimension 16x112x112.

The results I'm able to achieve are:
TOP1: 55.732
TOP5: 76.921

can you help me? thank you.

**Comment by fmassa** [comment]
@omedivad how many GPUs are you using for training? If I remember correctly, we used 64 GPUs.
You might need to adapt the learning rate if you change the number of GPUs.

Also, ccing @bjuncek if he has any more insights

**Comment by omedivad** [comment]
@fmassa I'm not training, I'm just using the torchvision pretrained model.

**Comment by bjuncek** [comment]
@omedivad this could be to dataset snapshot and dataset preparation. We evaluated on 19271 videos, rescaled to 480p on the shorter side at 15 fps single crop center transform at evaluation stage. I evaluated the same model on data that was downsampled to 140p along the shorter side and performance of that was much lower? 

How do you process your dataset?

**Comment by omedivad** [comment]
@bjuncek found it! I've rescaled videos to 128x171 as implemented in the vision/references/train.py code and described in the [paper](https://arxiv.org/abs/1711.11248) linked in torchvision video classification page. The resampling to 15 fps, and the centered single crop 112x112 are the same. Thank you very much! Did you rescale to 480p on the shorter side in training too?

**Comment by bjuncek** [comment]
No, so what I meant is that the raw videos (before the training ever begins) were stored in "higher" resolution then normally (I believe official Kinetics script squashes them to 256 on the shorter side).

Sorry for the confusion.

**Comment by omedivad** [comment]
Yes, the videos in my data snapshot have the shorter side 256 long. 
The problem could be this one. Thank you.
